can_tools/scrapers/official/NC/nc_vaccine.py

```diff
@@ -26,6 +26,12 @@ class NCVaccine(TableauDashboard):
         "AGG(Calc.Fully Vaccinated)-alias": variables.FULLY_VACCINATED_ALL,
     }
 
+    def fetch(self):
```

Comment:
our native `TableauDashboard` class was throwing errors that I couldn't quickly track down. The `TableauScraper` module did not have the same problems so I switched the `fetch()` method to use it instead.